esp/esdllib/README.md

```diff
@@ -546,33 +648,20 @@ Each of these parameters gives the script access to some information about the c
 
 * `$clientversion` : The client interface version specified in the incoming request.
 * `$espUserName` : The username of the user authenticated to run the current transaction.
 * `$espUserPeer` : The IP address of host originating the current transaction.
 * `$espUserRealm` : The 'Company' or parent entity to the currently authenticated user.
-* `$espUserStatus` : The status of the currently authenticated user (eg *Active*, *Suspended*, etc., as defined by the security plugin and back end in use)
+* `$espUserStatus` : The status of the currently authenticated user (eg *Active*, *Suspended*, etc., as defined by the security plugin and backend in use)
+* `$espTransactionID` : The transaction ID for the current in-flight transaction.
 
-In addition to the standard parameters above, any *Secure User Settings* configured as part of your service deployment are made available as parameters for you to declare and use in scripts. In configmgr, these settings are under ESP Process|ESP Service Bindings|Secure User Setting section. When configuring as an esdl application with a yaml file, it is in the authNZ.auth_map section
+In addition to the standard parameters above, any *Secure User Properties* configured as part of your service deployment are made available as parameters for you to declare and use in scripts. In configmgr, these settings are under *ESP Process | ESP Service Bindings | Feature Authentication* section. When configuring as an esdl application with a yaml file, it is in the *authNZ.auth_map* section
 
-### Variable Resolution
-Because of some implmentation details common implicit variables shouldn't be used in many XPath expressions as they aren't resolved correctly. For example you may want to extract a value from the incoming request using this command:
-
-```xml
-<es:set-value target="_ReferenceCode" select="soap:Body/$query/$request/Row/User/ReferenceCode"/>
-```
-
-In that example `$query` and `$request` won't be resolved and the XPath expression won't evaluate correctly. Instead you can use the `local-name()` XPath function as a workaround:
-
-```xml
-<es:set-value target="_ReferenceCode" select="soap:Body/*[local-name()=$query]/*[local-name()=$request]/Row/User/ReferenceCode"/>
-```
 ## Entry Points
 
-Generally, each entry point has default source and target nodes, which are the root nodes for read and write commands respectively. These defaults can be overridden with `@source` and `@target` attributes in the entry point, which we'll discuss here, and also using [source](#source) and [target](#target) [commands](#commands).
-
-At the end of each script
+Generally, each entry point has default source and target nodes, which are the root nodes for read and write operations respectively. These defaults can be overridden with `@source` and `@target` attributes in the entry point, which we'll discuss here, and also using [source](#source) and [target](#target) [commands](#commands).
```

Comment:
Consistency between *operation* and *command*?

Reply:
fixed